### PR TITLE
Enable dark theme

### PR DESCRIPTION
**Pull request by gjxo12**
1. setting theme color default or black
2. dark theme use default primary, accent tag

hello, I see web site in monitor, I think Dark mode may be required.

In terms of UI, only the theme was changed and the primary and accent settings were kept as default, but I will modify them if changes are necessary.

### CODE REVIEW

**Comment by luncliff** [comment]
Appreciate your contribution. I didn't read the recent release of MkDocs Material. Give me some days for review

**Review by luncliff — APPROVED** [review]

**Comment by luncliff** [comment]
Appreciate your contribution 👏 👏 👏

**Review by utilForever — COMMENTED** [review]
LGTM. Thnaks!
@luncliff Please don't merge before all reviewers are approved.

**Pull request merged**
